website/docs/plugins/ray_launcher.md

```diff
@@ -49,66 +49,65 @@ $ python your_app.py hydra/launcher=ray_aws --cfg hydra -p hydra.launcher
 _target_: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
 mandatory_install:
   hydra_version: 1.0.3
-  ray_version: 1.0.0
+  ray_version: 1.0.1.post1
   cloudpickle_version: 1.6.0
-  omegaconf_version: 2.1.0dev9
+  omegaconf_version: 2.1.0dev13
```

Comment:
How is this interpreted?
you are appending _version to the package name. is Ray parsing this?